docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,205 @@
+# Styling components
```

Comment:
@mihashco, @nawab-samsung would it be accurate to change the title to:

# Styling UI components with XAML

Reply:
Done